matlab/bart.m

```diff
@@ -1,27 +1,42 @@
 function [varargout] = bart(cmd, varargin)
-% BART	Call BART command from Matlab.
-%   [A, B] = bart('command', X, Y) call command with inputs X Y and outputs A B
+% BART Call BART command from Matlab.
+%   [varargout] = BART(cmd, varargin) to run given bart command (cmd) using the
+%   data arrays/matrices passed as varargin.
 %
+%   [A, B] = BART('command', X, Y) call command with inputs X Y and outputs A B
+%
+%   To output a list of available bart commands simply run "bart". To
+%   output the help for a specific bart command type "bart command -h".
+%
+% Parameters:
+%   cmd:        Command to run as string (including non data parameters)
+%   varargin:   Data arrays/matrices used as input
+%
+% Example:
+%   bart traj -h
+%   [reco] = bart('nuft -i traj', data) call nufft with inputs data and outputs reco
+%
+% Authors:
 % 2014-2016 Martin Uecker <uecker@med.uni-goettingen.de>
-% Edited for WSL, by Soumick Chatterjee <soumick.chatterjee@ovgu.de>
-
+% 2018 (Edited for WSL) Soumick Chatterjee <soumick.chatterjee@ovgu.de>
+% 2020 Martin Krämer <martin.kraemer@med.uni-jena.de>
 
+    % Check input variables
 	if nargin==0 || all(cmd==0)
```

Comment:
Should all(cmd==0) actually be isempty(cmd)?

Reply:
all(cmd==0) does seem to work, but is not as elegant, true. I have changed it to isempty(cmd)